### ISSUE TITLE
plot_basemap function seems to invert latitudes of underlying array

### ISSUE THREAD

**Issue by togeig** [CLIMADA-project/climada_python#12]
Hi all,
using the plot_basemap() function in the centroids base script seems to invert the underlying data grid with respect to the open streetmap basemap. Using the plot_basemap() function for Germany, I always end of with the OSM being upside-down, Baltic and North Sea in the South, with the scatter plot showing correctly. 

![test_DEU](https://user-images.githubusercontent.com/33146980/77176239-1368f000-6ac4-11ea-8ad4-3517f8f4c5ef.PNG)

Running the function line by line individually, I can avoid this behavior when replacing the CLIMADA plot_scatter() function by the geopandas plot() functionality, everything else being equal. Any experience with this behavior?
I obtain the same issue when running the Exposures notebook tutorial, see below:

```
import contextily as ctx
import matplotlib.pyplot as plt
# select the background image from the available ctx.sources
### standard example
ax = exp_templ.plot_basemap(buffer=30000, cmap='brg') # using open street map
#ax = exp_templ.plot_basemap(buffer=30000, url=ctx.sources.ST_WATERCOLOR, cmap='brg', zoom=9) # set image zoom

### my test example
test=exp_templ.copy()
test.to_crs(epsg=3857, inplace=True)
f, ax = plt.subplots(1, figsize=(8, 8))
ax = test.plot(axes=ax)
ctx.add_basemap(ax, zoom=13)
plt.show()
```

**Comment by gabrielaznar** [comment]
Hi togeig!
I think I solved this issue in one of the lasts commits (this one: https://github.com/CLIMADA-project/climada_python/commit/4720d76b738e64fe8950714ba2a3eaf13d9cae9c) and is included in the latest release 1.4.1. Could you check if you're using the latest climada? The contextily version used might also play a role...

**Comment by togeig** [comment]
Hi gabriela,
thanks for the update, this should indeed solve the issue. I just pulled the latest climada...
